src/twcc.c

```diff
@@ -0,0 +1,434 @@
+/**
+ * @file twcc.c Transport-wide Congestion Control (TWCC) Receiver Status
+ *
+ * https://tools.ietf.org/
+ * draft-holmer-rmcat-transport-wide-cc-extensions-01
+ *
+ * Copyright (C) 2025 Sebastian Reimers
+ */
+
+#include <re.h>
+#include <baresip.h>
+#include "core.h"
+
+
+enum {
+	TWCC_INTERVAL	= 100,	/**< ms                                */
+	TWCC_MAX_PAKETS = 100,	/**< Limit pakets per Feedback message */
+	TWCC_PKT_SIZE	= 1280, /**< max. Packet size in bytes         */
+};
+
+enum packet_state {
+	PACKET_NOT_RECEIVED = 0,
+	PACKET_RECEIVED,
+	PACKET_LARGE_DELAY /* Packet received, large or negative delta */
+};
+
+static const char uri[] = "http://www.ietf.org/id/"
+			  "draft-holmer-rmcat-transport-wide-cc-extensions-01";
+
+struct twcc_packet {
+	struct le le;
+	uint16_t tseq;
+	int32_t delta;
+	enum packet_state state;
+	uint64_t ts;
+};
```

Comment:
would it be possible to move this type and `enum packet_state` to libre ?

There might be useful to have functions to encode/decode to and from mbuf chunks/deltas ...

Reply:
Sounds good, will refactor this.